src/group/managed_group/mod.rs

```diff
@@ -309,9 +717,13 @@ impl From<MLSCiphertext> for MLSMessage {
 
 #[derive(Debug)]
 pub enum ManagedGroupError {
```

Comment:
Can we move the errors to a separate file to get the size of this one a little down?
Also, if this is supposed to be a public API integer errors would be nice for FFI.

Reply:
I moved the errors. As for the integers, we should come up with a comprehensive approach for the FFI. Integers don't make sense for Rust.